.coderabbit.yaml

```
@@ -0,0 +1,84 @@
+  request_changes_workflow: false
+  high_level_summary: true
+  high_level_summary_placeholder: '@coderabbitai summary'
+  auto_title_placeholder: '@coderabbitai'
+  review_status: true
+  poem: false
+  collapse_walkthrough: false
+  sequence_diagrams: true
+  path_filters:
+    - '!apps/classes/vendor/**'
+    - '!services/sumally-nx/**/gen/**'
+  path_instructions: [ ]
```

yamllint

[error] too many spaces inside empty brackets

(brackets)